### PR TITLE
Updated the DFS Base theme with code to match our private theme.

### PR DESCRIPTION
**Pull request by mortenson**
This PR updates the dfs_base theme with a lot of the generic SCSS/JS from our private repository. I wholesale copy+pasted a lot of this, so I'd like @briancwald to take a look over it and see if any of this is too specific to be shared by all of our new themes.

### CODE REVIEW

File: themes/dfs_base/scss/base/_font-settings.scss [hunk 1]
```diff
@@ -0,0 +1,68 @@
+//fonts used for fin
```
Comment (briancwald) [inline themes/dfs_base/scss/base/_font-settings.scss]: @mortenson I am not sure fonts in base are a good idea. This is specific to scenario

Comment (mortenson) [reply themes/dfs_base/scss/base/_font-settings.scss]: @briancwald How about keeping the icon fonts and dropping the others? I can see us using iconicons and fontawesome icons in future demos. I also plan to do code review today and clean up the comments which should remove instances of "fin". Sorry about that!

**Comment by briancwald** [comment]
Yes that would be my recommendation. Icons should move here, but not text fonts

**Comment by briancwald** [comment]
Looks good to me.

**Pull request merged**